src/main/kotlin/domain/Cell.kt

```diff
@@ -0,0 +1,23 @@
+package domain
+
+data class Cell(private var cellState: CellState = CellState(), private var mineNumber: MineNumber = MineNumber()) {
```

Comment:
지금처럼 한 줄로 길게 쓰셔도 상관은 없으나, 2개 이상인 경우에는 개행해주시는 것도 좋을 것 같아요
```suggestion
data class Cell(
    private var cellState: CellState = CellState(), 
    private var mineNumber: MineNumber = MineNumber()
) {
```